kv_store/Cargo.toml

```diff
@@ -0,0 +1,36 @@
+[package]
+name = "kv_store"
```

Comment:
We may need to rename `ed-kv-store` due to publishing constraints (there is already a kv-store`. Note also the use of `-` and not `_`.

I'm also happy to rename the `event-driven` and `event-driven-macros` dirs using their respective Cargo names.